### PR TITLE
Minor typo fixes

### LINKED ISSUE #528: Typing Erors in README
**Describe the issue**

There are some minor typing mistakes in README

**Would you like to work on the issue?**

Sure !

### PR DESCRIPTION
**Pull request by RajChowdhury240**
Fixes  #528

**Changes**:
Corrected Few minor spelling mistakes in README file

**Screenshot/s for the changes**: 
[Image](http://www.mediafire.com/view/9qx9hdppwq3njqv/preview.png)

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [ ] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Repository of Android app for the Neurolab Open Hardware platform.
 [![Mailing List](https://img.shields.io/badge/Mailing%20List-FOSSASIA-blue.svg)](https://groups.google.com/forum/#!forum/pslab-fossasia)
 [![Twitter Follow](https://img.shields.io/twitter/follow/pslabio.svg?style=social&label=Follow&maxAge=2592000?style=flat-square)](https://twitter.com/pslabio)
 
-This repository holds the Android app for the Neurolab Hardware. We are developing a neuro-device as a headband with integrated electronics. The goal of the Neurolab project is to create an easy to use open hardware measurement headset device for brain waves that can be plugged into an Android smartphone and a software application and enables us to understand our brains better.
+This repository holds the Android app for the Neurolab Hardware. We are developing a neuro-device as a headband with integrated electronics. The goal of the Neurolab project is to create an easy way to use open hardware measurement headset device for brain waves that can be plugged into an Android smartphone and a software application and enables us to understand our brains better.
```
Comment (dipu989) [inline README.md]: Please remove "way". "easy to use" is quoting that hardware can be used easily!

**Review by dipu989 — CHANGES_REQUESTED** [review]

**Comment by dipu989** [comment]
@Rajchowdhury420 Also, fill information in the template of the PR correctly. 
Take a look at https://github.com/fossasia/pslab-android/pull/1993.

**Review by RajChowdhury240 — COMMENTED** [review]
Fixed Minor typos

**Comment by RajChowdhury240** [comment]
@dipu989 check nah

**Comment by mariobehling** [comment]
Thanks. We need to move forward with the functionality though right now. Improving typos can also be done as part of improving functionality. Please help to focus on code.

**Review by jddeep — APPROVED** [review]
Thanks. Let us move from here. Please focus on features and code for now.

**Pull request merged**